R/pmcmc.R

```diff
@@ -66,6 +65,59 @@ pmcmc <- function(pars, filter, initial = NULL, control = NULL) {
 }
 
 
+##' Run a pMCMC, with sensible random number behaviour, but schedule
```

Comment:
This function doesn't seem to be running anything but only preparing? Makes the docs a bit confusing

Reply:
Both functions are documented on the same page :)